scripts/iam-tools/check_sso_http_status.py

```diff
@@ -0,0 +1,63 @@
+#!/bin/env python
+# vim: expandtab:tabstop=4:shiftwidth=4
+
+"""
+ This script is used to check the return code from /status.php IDP host and report to Zabbix.
+ It will report an issue to Zabbix if any received HTTP status codes != 200.
+"""
+
+from __future__ import print_function
+
+import os
+import requests
+import yaml
+
+# Reason: disable pylint import-error because our modules aren't loaded on jenkins.
+# pylint: disable=import-error
+from openshift_tools.monitoring.zagg_sender import ZaggSender
+
+
+class CheckStatus(object):
+    """ Class to check HTTP status code from IDP host. """
+
+    def __init__(self):
+        self.sso_status_code = None
+
+
+    @staticmethod
+    def check_http(ops_idp_host):
+        """ Check HTTP status code returned by the IDP host. """
+
+
+        url_path = 'https://' + ops_idp_host + '/status.php'
+        check_status = requests.get(url_path)
+        sso_status_code = check_status.status_code
+
+        return sso_status_code
+
+
+    def main(self):
+        """ Main function. """
+
+        zag = ZaggSender()
+        zabbix_key = "sso.http.not.reachable"
```

Comment:
we need this code to do 2 different checks. One of the actual service and one of the other (main) container in the pod. It'll take a little moving around (like moving the config file reader above the loop, etc., but this is basically what I had in mind:
```
checks = {
  "sso.service.not.reachable": yaml_config["idp_host"],
  "sso.container.not.reachable": "127.0.0.1:8443"
}
for zabbix_key, host in checks.iteritems():
   # ... do checks, pass in host to self.check_http()
```